pkg/internal/checkup/vmi/spec.go

```diff
@@ -262,6 +262,22 @@ func WithAffinity(affinity *corev1.Affinity) Option {
 	}
 }
 
+func WithReadinessFileProbe(fileName string) Option {
+	return func(vmi *kvcorev1.VirtualMachineInstance) {
+		var readinessProbeCommand = []string{"cat", fileName}
+		vmi.Spec.ReadinessProbe = &kvcorev1.Probe{
+			Handler: kvcorev1.Handler{
+				Exec: &corev1.ExecAction{
+					Command: readinessProbeCommand,
+				},
+			},
+			FailureThreshold:    30,
```

Comment:
Adding `TimeoutSeconds:      10,` helped reduce the number of queries GA makes, and gets to the Ready state faster.

Reply:
DONE